.github/workflows/mkdocs.yml

```diff
@@ -1,20 +1,26 @@
 name: Publish docs
 on:
   workflow_dispatch:
+  workflow_run:
+    workflows:
+      - Pytest
+    types:
+      - completed
+    branches:
+      - dev
   push:
     branches:
       - dev
     paths:
-      - 'benefits/**'
       - 'docs/**'
-      - 'tests/pytest/**'
       - 'mkdocs.yml'
       - '.github/workflows/mkdocs.yml'
 
 jobs:
   docs:
     name: Publish docs
     runs-on: ubuntu-latest
+    if: github.event.workflow_run == null || github.event.workflow_run.conclusion == 'success'
```

Comment:
@afeld does this syntax look correct for the case where it is not `workflow_run`? 

Another possibility is `github.event_name != 'workflow_run'`, not sure if that's better / more correct?

Reply:
Hmm. I haven't set up dependencies between GitHub Actions workflows before, so would need to do some research myself. For starters, seems like the `push` trigger above is redundant with the `workflow_run`, since the latter would be coming from the mkdocs one that already happens on `push`, right? If it works, it works 🤷 

I suggest we don't spend _too_ much time finagling the deployment; worst case, we can either:

- Smush the mkdocs stuff into the same workflow/job as pytest
- Direct folks to download the artifact from the [GitHub Actions runs](https://github.com/cal-itp/benefits/actions/workflows/tests-pytest.yml?query=branch%3Adev)

and call it a day.

Reply:
Yeah agreed. I was more talking about the syntax of the `if` specifically, but I think this one now works as intended.

The `push` is still necessary to re-publish the docs site when only docs change (like fixing a typo or adding a new page/section).

`workflow_run` makes the docs site re-publish when python code changes and `coverage` is updated in the `dev` branch.